<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitor</class>
 <widget class="QWidget" name="Monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>579</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(164, 0, 0);
background-color: rgb(233, 185, 110);</string>
  </property>
  <widget class="QWidget" name="gridWidget" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>8</y>
     <width>871</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="styleSheet">
         <string notr="true">color: rgb(252, 249, 249);
background-color: rgb(78, 154, 6);
alternate-background-color: rgb(186, 189, 182);</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="closeButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(252, 249, 249);
background-color: rgb(78, 154, 6);
alternate-background-color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTextBrowser" name="textBrowser1">
      <property name="styleSheet">
       <string notr="true">color: rgb(252, 249, 249);
background-color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QTextBrowser" name="textBrowser2">
      <property name="styleSheet">
       <string notr="true">color: rgb(252, 249, 249);
background-color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLabel" name="label4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextBrowser" name="textBrowser4">
        <property name="styleSheet">
         <string notr="true">color: rgb(252, 249, 249);
background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTextBrowser" name="textBrowser3">
        <property name="styleSheet">
         <string notr="true">color: rgb(252, 249, 249);
background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTextBrowser" name="processBrowser">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1999999</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(252, 249, 249);
background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
